backend/conftest.py

```diff
@@ -1,9 +1,23 @@
+import os
```

Comment:
can we have this on application-level, not for the whole backend...

as the app will grow it may give us headaches?